nms/app/packages/magmalte/app/views/subscriber/SubscriberOverview.js

```diff
@@ -221,30 +223,68 @@ function SubscriberInternal(props: WithAlert) {
       width: 200,
     },
   ];
+  const [tokenList, setTokenList] = useState([]);
 
-  const subscribersIds = Array.from(
-    new Set([...Object.keys(subscriberMap), ...Object.keys(sessionState)]),
-  );
+  function handleSubscriberQuery(networkId, q) {
+    return new Promise(async (resolve, reject) => {
+      try {
+        const page = 100 < q.page * q.pageSize ? 100 / q.pageSize : q.page;
+        const subscriberQuery = await MagmaV1API.getLteByNetworkIdSubscribersV2(
+          {
+            networkId,
+            pageSize: 10,
+            pageToken: page === 0 ? '' : tokenList[page - 1],
```

Comment:
Can you help me understand how this token list logic works ? Should we disable the last page option ? We can only traverse sequentially right?